service/src/test/java/bio/terra/workspace/service/resource/controlled/cloud/azure/vm/CreateAzureVmStepTest.java

```diff
@@ -180,6 +192,19 @@ public void setup() {
     // get network mocks
     when(mockNetworkManager.networks()).thenReturn(mockNetworks);
     when(mockNetworks.getByResourceGroup(anyString(), anyString())).thenReturn(mockNetwork);
+    when(mockNetworkManager.networkInterfaces()).thenReturn(mockNis);
```

Comment:
I feel like there should be a less tedious way to do this. Maybe using a fake instance instead of a mock would help. It looks like this requires all these calls to come in a specific order too.

Reply:
yea that's a good point...maybe will be a better pattern for new development in the future